Userland/Applications/Browser/DownloadWidget.cpp

```diff
@@ -69,20 +69,26 @@ DownloadWidget::DownloadWidget(const URL& url)
     m_browser_image->load_from_file("/res/graphics/download-animation.gif"sv);
     animation_container.add_spacer().release_value_but_fixme_should_propagate_errors();
 
-    auto& source_label = add<GUI::Label>(String::formatted("From: {}", url).release_value_but_fixme_should_propagate_errors());
+    auto& source_label = add<GUI::Label>(String::formatted("File: {}", m_url.basename()).release_value_but_fixme_should_propagate_errors());
     source_label.set_text_alignment(Gfx::TextAlignment::CenterLeft);
     source_label.set_fixed_height(16);
+    source_label.set_text_wrapping(Gfx::TextWrapping::DontWrap);
 
     m_progressbar = add<GUI::Progressbar>();
     m_progressbar->set_fixed_height(20);
 
     m_progress_label = add<GUI::Label>();
     m_progress_label->set_text_alignment(Gfx::TextAlignment::CenterLeft);
     m_progress_label->set_fixed_height(16);
+    m_progress_label->set_text_wrapping(Gfx::TextWrapping::DontWrap);
 
-    auto& destination_label = add<GUI::Label>(String::formatted("To: {}", m_destination_path).release_value_but_fixme_should_propagate_errors());
+    DeprecatedString m_to_label_path = m_destination_path;
```

Comment:
The prefix `m_` indicates that the variable is a *M*ember of the class. This is a common pattern, because surprisingly often we deal with the same thing as a parameter *and* want to access the own member variable, and this avoids shadowing.

Therefore, please don't call a local variable `m_…` unless perhaps in very extremely rare situations.

Also, why "to"? The actual `GUI::Label` variable is called `destination_label`, so `destination_label_path` seems more obvious.